app/services/sandbox_seed_data/delivery_partners.rb

```diff
@@ -0,0 +1,27 @@
+module SandboxSeedData
+  class DeliveryPartners < Base
+    NUMBER_OF_RECORDS = 50
+
+    def plant
+      return unless plantable?
+
+      log_plant_info("delivery partners")
+
+      NUMBER_OF_RECORDS.times do
+        delivery_partner = create_delivery_partner
+
+        log_seed_info(delivery_partner.name, colour: Colourize::COLOURS.keys.sample)
+      end
+    end
+
+  private
+
+    def create_delivery_partner
```

Comment:
maybe not in this PR but we need to go through every lead provider across every cohort, but happy to wait for the dp ticket to tackle this